examples/generative/corrdiff/training/training_loop.py

```diff
@@ -262,6 +283,8 @@ def training_loop(
                 )
                 training_stats.report("Loss/loss", loss)
                 loss.sum().mul(loss_scaling / batch_gpu_total).backward()
+                all_loss.append(loss.sum().mul(loss_scaling / batch_gpu_total).item())
```

Comment:
Loss reduction is done twice. Can we instead have
`loss = loss.sum().mul(loss_scaling / batch_gpu_total)
loss.backward()
all_loss.append(loss.time())
`